samples/javascript/04.live-share-react/src/App.jsx

```diff
@@ -0,0 +1,85 @@
+import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+import {
+    AzureProvider,
+    LiveShareProvider,
+} from "@microsoft/live-share-react";
+import { inTeams } from "./utils/inTeams";
+import { useRef } from "react";
+import {
+    ExampleLivePresence,
+    ExampleSharedMap,
+    ExampleLiveState,
+    ExampleSharedState,
+    ExampleMediaSynchronizer,
+    ExampleLiveEvent,
+    ExampleLiveCanvas,
+    ExampleLiveTimer,
+    EXAMPLE_SHARED_MAP_KEY,
+} from "./components";
+import { TeamsClientLoader } from "./components/TeamsClientLoader";
+import { LiveShareHost } from "@microsoft/teams-js";
+import { TestLiveShareHost } from "@microsoft/live-share";
+import { SharedMap } from "fluid-framework";
+
+const localConnection = {
+    type: "local",
+    tokenProvider: new InsecureTokenProvider("", {
+        id: "123",
+    }),
+    endpoint: "http://localhost:7070",
+};
+const azureClientOptions = {
+    connection: localConnection,
+};
+const host = inTeams() ? LiveShareHost.create() : TestLiveShareHost.create();
+
+export default function App() {
+    // set to false to use AzureClient Fluid container
+    const shouldUseLiveShare = useRef(true);
+    if (shouldUseLiveShare.current) {
+        return (
+            <TeamsClientLoader>
+                <LiveShareProvider
+                    joinOnLoad={true}
```

Comment:
As discussed, consider adding flow for `false` case and to show how dynamic DDS loading looks like.

Reply:
The current sample already shows how dynamic DDS loading works. I will handle the false case as discussed, but can you please clarify what you mean by `show how dynamic DDS loading looks like`?

Reply:
Done